components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import React from 'react';
+import SlideCard from './SlideCard';
+import { EmblaOptionsType } from 'embla-carousel';
+import {
+    PrevButton,
+    NextButton,
+    usePrevNextButtons
+} from './EmblaCarouselArrowButton';
+import useEmblaCarousel from 'embla-carousel-react';
+
+type PropType = {
+    slides: number[];
+    options?: EmblaOptionsType;
+};
+
+const EmblaCarousel = (props: PropType) => {
+    const slideNames = ['Арија', 'Реа', 'Фреја', 'Јоги', 'Дона'];
+    const { slides, options } = props;
```

Comment:
- `slides` variable / prop is not used anywhere. You're mapping them on `slideNames`, which are hardcoded in the component. That makes the entire declaration on `page.tsx` (Homepage) redundant. 
- `options` variable can be hardcoded into this component instead of being passed down from the parent.

This will be changed in the future to retrieve data from an API, but you can do a little bit of cleanup to prepare for that. Maybe it's best if you made some `json` dummy data which will contain the images info and just pass the `url` to this component which will fetch the url and render the component accordingly.

Reply:
example json:
```
{
    "images": [
        {
            "id": "1",
            "name": "foo",
            "url": "../foo.jpg"
        },
        {
            "id": "2",
            "name": "bar",
            "url": "../bar.jpg"
        }
    ]
}
```